content/_index.zh.html

```diff
@@ -0,0 +1,224 @@
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+
+<div class="homepage">
+  <section class="hero">
+    <div class="grid-container">
+      <img src="/images/homepage/home hero blue_v2@2x.png" alt="Hero image">
+      <div class="text-container">
+        <h1>简化加密</h1>
+        <p>NavCoin是一种易于使用的数字货币，由其社区运营。在网络经济中，可编程货币降低了做生意的&nbsp;成本</p>
```

Comment:
```suggestion
        <p>NavCoin是一种易于使用的数字货币，由其社区运营。在网络经济中，可编程货币降低了做生意的&nbsp;成本。</p>
```

Reply:
done and &nbsp; code in Chinese files is deleted